src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -42,7 +42,8 @@ def convert_tf_weight_name_to_pt_weight_name(tf_name, start_prefix_to_remove="")
         - transpose: boolean indicating whether TF2.0 and PyTorch weights matrices are transposed with regards to each
           other
     """
-    tf_name = tf_name.replace(":0", "")  # device ids
+    tf_name = tf_name.replace(":0", "")  # device ids]
+
```

Comment:
No need for this change I think
```suggestion
    tf_name = tf_name.replace(":0", "")  # device ids
```